_includes/specials/policy/en/04-feerate-estimation.md

```diff
@@ -7,4 +7,91 @@
   if uncertain about anything, just do what seems best and harding will edit
 -->
 
-FIXME:glozow
+Last week, we explored techniques for minimizing the fees paid on a
+transaction given a feerate. But what should that feerate be? Ideally,
+as low as possible to save money, but high enough to secure a spot in
+a block suitable for the user's time preference.
+
+The goal of _fee estimation_ is to translate a target timeframe for
+confirmation to a minimal feerate the transaction should pay.
+
+One complication of fee estimation is the irregularity of block space
+production. Let's say a user needs to pay a merchant within one hour
+to receive their goods. The user may expect a block to be mined every
+10 minutes, and thus aim for a spot within the next 6 blocks. However,
+it's entirely possible for one block to take 45 minutes to be found.
+Fee estimators must translate between a user’s desired urgency or
+timeframe (something like “I want this to confirm by the end of the
+work day”) and a window of block space (a number of blocks). Many fee
+estimators address this challenge by denominating confirmation targets
+in the number of blocks in addition to time.
+
+With no information about transactions prior to their confirmation,
+one can build a naive fee estimator that uses historical data about
+what transaction feerates tend to land in blocks. As this estimator is
+blind to the transactions awaiting confirmation in mempools, it would
+become very inaccurate with unexpected changes in blockspace demand
+and the occasional long block interval. Its other weakness is its
+reliance on information controlled wholly by miners, who would be able
+to drive feerates up by including fake high-feerate transactions in
+their blocks.
+
+Fortunately, the market for blockspace is not a blind auction. We
+mentioned in our first post that keeping a mempool and participating
+in the peer-to-peer transaction relay network allows a node to see
+what users are bidding. The Bitcoin Core fee estimator also uses
+historical data to calculate the likelihood of a transaction at
+feerate `f` confirming within `n` blocks, but specifically tracks the
+height at which the node first sees a transaction and when it
+confirms.  This method works around activity that happens outside the
+public fee market by ignoring it. If miners always include several
+high-feerate transactions in their own blocks, this fee estimator
+isn't skewed because it only uses data from transactions that were
+publicly relayed prior to confirmation.
```

Comment:
This is a great point, and maybe to clarify slightly:
```suggestion
publicly relayed prior to confirmation. (A miner won't _broadcast_ a fake high-feerate transaction because it could be mined by a competing miner.)
```